scipy/signal/tests/test_bsplines.py

```diff
@@ -65,6 +65,49 @@ def test_spline_filter(self):
         assert_allclose(bsp.spline_filter(data_array_real, 0),
                         result_array_real)
 
+    def test_spline_filter_complex(self):
+        # Test the complex branch : restore the test which was removed
+        # in https://github.com/scipy/scipy/pull/17455 but should not
+        # have been. cf. https://github.com/scipy/scipy/pull/9208/
```

Comment:
IMO everything after "Test the complex branch" is better suited for the commit message rather than a test comment. (712f75312a3e7e2a63e68beafa24296cebeb55bd)

Reply:
OK, done